src/rgw/store/dbstore/tests/dbstore_tests.cc

```diff
@@ -687,8 +687,7 @@ TEST_F(DBStoreTest, GetObject) {
 TEST_F(DBStoreTest, GetObjectState) {
   struct DBOpParams params = GlobalParams;
   int ret = -1;
-  RGWObjState state;
-  RGWObjState *s = &state;
+  RGWObjState *s;
```

Comment:
done.

Reply:
um, I was just suggesting the cosmetic change to write RGWObjState* for "pointer to RGWObjState"--not to remove it's assignment from state--most likely, not assigning it will cause a crash?

Reply:
@soumyakoduri

Reply:
sorry, I typed that badly, it wasn't supposed to have a semicolon

Reply:
's' is now initialized to point to RGWObjState stored as part of DBObject in get_obj_state(). Hence it need not be initialized to state here. 
Also I realized there are couple of other places to change to 'RGWObjState* '.. Will update.